src/core/sys/posix/time.d

```diff
@@ -408,9 +408,7 @@ else version( CRuntime_Musl )
     enum CLOCK_REALTIME = 0;
     enum CLOCK_PROCESS_CPUTIME_ID = 2;
     enum CLOCK_THREAD_CPUTIME_ID = 3;
-    enum CLOCK_MONOTONIC_RAW = 4;
     enum CLOCK_REALTIME_COARSE = 5;
-    enum CLOCK_MONOTONIC_COARSE = 6;
```

Comment:
Oh, these are removed because they're already declared in a `version(linux)` block at the top.